combat-trainer.lic

```diff
@@ -1999,6 +2036,62 @@ class GameState
     @attack_overrides[weapon_skill]
   end
 
+  def set_currently_whirlwinding(setting)
+    @currently_whirlwinding = setting
+  end
+
+  def currently_whirlwinding?
```

Comment:
Likewise, this method isn't necessary - the only different would be you wouldn't have a `?` if you used the one provided by `attr_accessor`